code/game/machinery/primitiveprocessor.dm

```diff
@@ -0,0 +1,157 @@
+//here is the chopping block.
+//it can be created from 2 wooden planks
+//place this on a table or else you will build it in your hands, which places at your feet!
+/obj/item/chopping_block
+	name = "chopping block"
+	desc = "A primitive version that cavemen used to use for processing food. Yuck!"
```

Comment:
My god this grammar. Also fix the lines on 22 and 75,

"A primitive version that was used by cavemen for processing food. Yuck!"